pkg/operation/botanist/component/istio/charts/istio/istio-ingress/templates/deployment.yaml

```diff
@@ -35,12 +41,16 @@ spec:
         image: {{ .Values.image }}
         imagePullPolicy: IfNotPresent
         securityContext:
-          allowPrivilegeEscalation: false
+          # Safe since 1.22: https://github.com/kubernetes/kubernetes/pull/103326
```

Comment:
Hmm here's the same comment, but no check if k8s version is higher than 1.22. Is that ok?